.cirrus.yml

```diff
@@ -132,3 +141,18 @@ promote_task:
     folder: ${CIRRUS_WORKING_DIR}/.m2/repository
   script: cirrus_promote_maven
   cleanup_before_cache_script: cleanup_maven_repository
+
+build_and_release_sonar_secrets_dotnet_task:
+  depends_on:
+    - promote
```

Comment:
Discussed offline: the other tasks fail if this task does not run last.
There's an open card and it's under investigation why this happens.
For this PR it will be left as is.

Reply:
Update: Seems to work. Will update here.